index.bs

```diff
@@ -502,9 +522,9 @@ After the step:
 
 Add the step:
 
-1. [=Set Attribution Reporting headers=] with |req|'s [=request/header list=]
-    and <a>this</a>'s <a href=#xmlhttprequest-eligibility>Attribution Reporting
+1. Set |req|'s [=request/Attribution Reporting eligibility=] to <a>this</a>'s <a href=#xmlhttprequest-eligibility>Attribution Reporting
     eligibility</a>.
+1. [=Set Attribution Reporting headers=] with |req|.
```

Comment:
The `Set Attribution Reporting headers` algorithm has two parameters: this "invocation" is missing the context origin.